pkg/k8s/watchers/service.go

```diff
@@ -552,20 +564,28 @@ func (k *K8sServiceWatcher) addK8sSVCs(svcID k8s.ServiceID, oldSvc, svc *k8s.Ser
 		// are no longer in the updated service and delete them in the datapath.
 		oldSVCs, err := k.datapathSVCs(oldSvc, endpoints)
 		if err != nil {
-			scopedLog.WithError(err).Error("Error while evaluating datapath services for old service")
+			scopedLog.Error("Error while evaluating datapath services for old service", logfields.Error, err)
 			return
 		}
 		oldSVCMap := hashSVCMap(oldSVCs)
 
 		for svcHash, oldSvc := range oldSVCMap {
 			if _, ok := svcMap[svcHash]; !ok {
 				if found, err := k.svcManager.DeleteService(oldSvc); err != nil {
-					scopedLog.WithError(err).WithField(logfields.Object, logfields.Repr(oldSvc)).
-						Warn("Error deleting service by frontend")
+					scopedLog.Warn(
+						"Error deleting service by frontend",
+						logfields.Error, err,
+						logfields.Object, oldSvc,
+					)
 				} else if !found {
-					scopedLog.WithField(logfields.Object, logfields.Repr(oldSvc)).Warn("service not found")
+					scopedLog.Warn(
+						"service not found",
+						logfields.Object, oldSvc,
```

Comment:
nit: previously this was `logfields.Repr(oldSvc)` - what's the difference?

Reply:
To the user: none :tada: 
To the CPU: less calculations :tada: